packages/mwp-api-proxy-plugin/src/util/receive.js

```diff
@@ -203,17 +202,21 @@ export const makeLogResponse = request => ([response, body]) => {
 			errorMessage = JSON.stringify(info.errors[0]) || body;
 		} catch (err) {
 			// probably not JSON, could be an HTML response
-			errorMessage = 'REST API error';
+			const titleContent = /<title>(.+?)<\/title>/.exec(body);
```

Comment:
Is the descriptive error message in the `title` something we are doing, or is it something fastly does?

Mostly, just curious what sort of errors we are capturing with title tags.

Reply:
It's something the REST API does - e.g. for those 503 errors, the body of the response is:

```
<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
 "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html>
  <head>
    <title>503 Request body sent more than once</title>
  </head>
  <body>
    <h1>Error ..."   
```

Reply:
p.s. I think this specific 503 error _might_ appear if we're sending a request body with a GET request - I need to follow up on that

Reply:
Cool.